### PR TITLE
Docs/readme

### PR DESCRIPTION
**Pull request by mdip226**

### CODE REVIEW

**Comment by briandrawert** [comment]
I was not able to successfully follow the readme.  See #47

File: README.md [hunk 1]
```diff
@@ -1,12 +1,36 @@
+# StochSS-Compute
+
+With StochSS-Compute, you can run GillesPy2 simulations on your own server. Results are cached and anonymized, so you
+can easily save and recall previous simulations.
```
Comment (ethangreen-dev) [inline README.md:4]: I would replace this with something for generic (as GillesPy2 is just the start). Maybe something like this "A distributed compute delegation server for the StochSS family of stochastic simulators."?

File: README.md [hunk 2]
```diff
@@ -1,12 +1,36 @@
...
+```
+git clone https://github.com/StochSS/stochss-compute.git
+cd stochss-compute
+pip install -r requirements.txt
+python3 app.py
+```
+- Stochss-compute is now running on localhost:1234.
+- Check out some example notebooks for more instructions on usage.
+- Dask compute cluster configuration parameters can be passed to `app.py`, see the [documentation](https://github.com/StochSS/stochss-compute/blob/dev/stochss_compute/api/delegate/dask_delegate.py#L20).
+
```
Comment (ethangreen-dev) [inline README.md:21]: We should probably add two sections: How to set Dask up, and how to best configure stochss-compute. We can chat strategies about the latter.

File: README.md [hunk 3]
```diff
@@ -118,8 +100,12 @@ class ToggleSwitch(gillespy2.Model):
...
 model = ToggleSwitch()
 
-# Run the model on a stochss-compute server instance running on localhost.
-results = RemoteSimulation.on(ComputeServer("127.0.0.1", port=1234).with_model(model).run()
+# Run the model on a stochss-compute server instance running on localhost. 
+# The default port is 1234, but will depend on how you choose to set it up.
+results = RemoteSimulation.on(ComputeServer("127.0.0.1", port=1234)).with_model(model).run()
+
+# Wait for the simulation to finish.
+results.wait()
```
Comment (ethangreen-dev) [inline README.md:108]: `results.wait()` will wait for the job to finish (but not get the results), `results.resolve()` will wait for the job to complete and grab the results from the server.

File: stochss_compute/api/delegate/dask_delegate.py [hunk 4]
```diff
@@ -59,9 +59,8 @@ def __init__(self, delegate_config: DaskDelegateConfig):
             if self.delegate_config.kube_cluster is not None:
                 self.client = Client(self.delegate_config.kube_cluster)
                 print(self.delegate_config.kube_cluster)
-            # TODO what happens if user is not using kubernetes?
             else:
-                self.client = Client(self.delegate_config.cluster_address)
+                self.client = Client()
```
Comment (ethangreen-dev) [inline stochss_compute/api/delegate/dask_delegate.py:63]: Revert this change. Issue [#47](https://github.com/StochSS/stochss-compute/issues/47) was fixed with PR #49.

**Review by ethangreen-dev — CHANGES_REQUESTED** [review]
Fantastic work so far, just needs a couple additions / changes and it'll be perfect.

File: README.md (hunk 2 above)
Comment (mdip226) [reply README.md:34]: I agree, def

File: README.md (hunk 1 above)
Comment (mdip226) [reply README.md:4]: I was gonna do that but then I just figured to wait until that's fully implemented

Comment (mdip226) [reply README.md:4]: but yeah, i agree with what you said

File: README.md (hunk 3 above)
Comment (mdip226) [reply README.md:121]: ah, gotcha.

File: README.md (hunk 2 above)
Comment (mdip226) [reply README.md:36]: I'm just going to make a separate notebook for this

**Review by briandrawert — CHANGES_REQUESTED** [review]
When I walked through the README, I got the error in #58

**Comment by briandrawert** [comment]
Merge #52 before merging this PR

**Review by briandrawert — APPROVED** [review]

**Pull request merged**